multicluster/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,13 @@
+// Copyright YEAR Antrea Authors
```

Comment:
Remove this file.
I know you are using kube-builder to generate these codes automatically, but we changed and re-organized the project a lot, please make sure you remove unnecessary files and changes.